### ISSUE TITLE
Greek characters in circuit drawings creating alignment problems

### ISSUE THREAD

**Issue by abbycross** [Qiskit/documentation#187]
### URL to the relevant documentation

Once the page is live, ~the URL will be https://docs.quantum-computing.ibm.com/verify/simulate-with-qiskit-primitives (will merge [in this PR](https://github.com/Qiskit/documentation/pull/184))~ See https://docs.quantum.ibm.com/build/circuit-visualization#draw-a-quantum-circuit

### Please describe the UI problem.

Circuit drawings on this page aren't completely aligned correctly. Under Parameterized circuit with Estimator, the drawing in the old Qiskit docs shows this
<img width="204" alt="Screenshot 2023-10-17 at 1 11 00 PM" src="https://github.com/Qiskit/documentation/assets/28769477/10373feb-ea49-4242-ba3d-c4b8eb0a2073">

But in our docs it shows as this 
<img width="256" alt="Screenshot 2023-10-17 at 1 12 03 PM" src="https://github.com/Qiskit/documentation/assets/28769477/e0ceda81-f12a-4b91-876b-d42f8f89fa64">

The problem seems to be the theta character.

The problem is similar in the analogous Sampler section
<img width="369" alt="Screenshot 2023-10-17 at 1 12 52 PM" src="https://github.com/Qiskit/documentation/assets/28769477/c920af58-93a5-4630-b8f5-d2e0b49b386a">

**Comment by javabster** [comment]
This is an issue bc ibm plex doesnt have a font for greek characters, so in browser it defaults to a different font which throws off the spacing

**Comment by frankharkins** [comment]
From Abby's screenshot, it seems the old Qiskit docs used a fallback with the correct spacing. We might be able to find that font and use it here.

**Comment by Eric-Arellano** [comment]
This is now fixed through changes in closed source.

<img width="250" alt="Screenshot 2024-05-30 at 3 29 57 PM" src="https://github.com/Qiskit/documentation/assets/14852634/0def34e3-4051-443e-81ac-8c915095da24">

There were two issues:

1. For QuantumCircuit.draw(), Qiskit returns both HTML and plain text. The app was preferring the HTML, but its formatting doesn't work well with the web app. We now use the plain text
2. Next.js was using a fallback font of Arial for characters that aren't in IBM Plex Mono. We turned that off and now fallback to a monospace font.
